pkgs/top-level/all-packages.nix

```diff
@@ -23481,12 +23481,6 @@ in
     inherit (xorg) libxcb;
   };
 
-  qvim = lowPrio (callPackage ../applications/editors/vim/qvim.nix {
```

Comment:
It's probably worth adding an entry to `aliases.nix`.

Reply:
Good idea - done.